src/BenchmarkDotNet/Validators/BenchmarkClassValidator.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using BenchmarkDotNet.Extensions;
+using BenchmarkDotNet.Running;
+
+namespace BenchmarkDotNet.Validators
+{
+    public class BenchmarkClassValidator: IValidator
+    {
+        public static readonly IValidator Default = new BenchmarkClassValidator();
+        
+        public bool TreatsWarningsAsErrors => false;
```

Comment:
this should return true => if `T` is private we can't benchmark it, so it's an error (not warning)